packages/fuchsia_remote_debug_protocol/examples/list_vms_and_flutter_views.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:core';
+
+import 'package:fuchsia_remote_debug_protocol/fuchsia_remote_debug_protocol.dart';
+import 'package:fuchsia_remote_debug_protocol/logging.dart';
+
+/// Runs through a simple usage of the fuchsia_remote_debug_protocol library:
+/// connects to a remote machine at the address in argument 1 (interface
+/// optional for argument 2) to list all active flutter views and Dart VM's
+/// running on said device. Note that this uses an SSH config file (optional,
```

Comment:
nit: strike "note that", it doesn't add anything.

Reply:
Fixed.